learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -0,0 +1,162 @@
+```
+layout: getting-started-with-ballerina-left-nav-pages
+title: Getting Started with Ballerina
+description: Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+keywords: ballerina, programming language, ballerina packages, getting started
+permalink: /learn/getting-started-with-ballerina
+active: getting-started-with-ballerina
+intro: Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+---
+```
+
+Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- A command terminalA text editor.
+- [VSCode](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed is our preferred choice, and it has good support for Ballerina.
+- A Ballerina installation. Follow the steps in [Installing Ballerina](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/).
+- Some experience with at least one programming language. 
```

Comment:
```suggestion
- Some experience with at least one programming language. 
```
Please check the comments added in https://github.com/ballerina-platform/ballerina-dev-website/pull/3369 for the GraphQL guide.

Reply:
I checked. There is nothing related to this line in that PR.